packages/react-strict-dom/tests/html-test.native.js

```diff
@@ -162,6 +162,20 @@ describe('<html.*>', () => {
       expect(root.toJSON()).toMatchSnapshot();
     });
 
+    test('"caretColor" red', () => {
+      const root = create(<html.input style={{ caretColor: 'red' }} />);
+      expect(root.toJSON()).toMatchSnapshot();
+    });
+
+    test('"caretColor" on unsupported values', () => {
+      create(<html.input style={{ caretColor: 'inherit' }} />);
+      expect(console.warn).toHaveBeenCalledWith(
+        expect.stringContaining(
+          'unsupported style value in "caretColor:inherit"'
+        )
+      );
+    });
+
```

Comment:
I moved these tests to the css-test.native.js file, as the feature didn't need a React tree for testing.